book_models/models/book.py

```diff
@@ -8,7 +8,7 @@ class Book:
     """Basic model for a book."""
     def __init__(
             self,
-            author: str,
+            authors: str,
```

Comment:
Perhaps it should be better for this field to be a list of strings.

Reply:
okay i'll do that

Reply:
Just for reference, don't start an entire chat here, only reply with questions/other suggestions/or the actual changes.

Reply:
The required changes are done please check my commit 063bb936cca0561a2c408cff1a0594093e4f375e